### ISSUE TITLE
[dataclasses] docstrings for pybindings (Trac #1443)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1435]
Add docstrings to pybindings where appropriate.  Such as I3MCTreePhysicsLibrary.cxx

Migrated from https://code.icecube.wisc.edu/ticket/1443
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:57", 
    "description": "Add docstrings to pybindings where appropriate.  Such as I3MCTreePhysicsLibrary.cxx", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067237750774", 
    "component": "combo core", 
    "summary": "[dataclasses] docstrings for pybindings", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-11-24T20:33:57", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160420T18:52:16: olivas changed milestone from "March 2016" to "April 2016"

**Comment by icecube-trac** [comment]
Trac update at 20160517T02:42:51: olivas changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20160815T15:36:16: 
* jlanfranchi changed owner from "olivas" to "jlanfranchi"
* jlanfranchi changed status from "accepted" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20170220T18:23:15: 
* olivas commented:

> Anticipating Giuliano's response, I'm reassigning this to him. :)


* olivas changed owner from "jlanfranchi" to "gmaggi"

**Comment by icecube-trac** [comment]
Trac update at 20170331T19:26:19: 
* olivas changed status from "assigned" to "accepted"
* olivas changed owner from "gmaggi" to "olivas"
* olivas changed milestone from "May 2016" to "June 2017"

**Comment by icecube-trac** [comment]
Trac update at 20170630T18:50:35: 
* olivas commented:

> In changeset 156891:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="156891"
> added docstrings for mctree physics library.  closes #1435.
> ```


* olivas changed resolution from "" to "fixed"
* olivas changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:57: 
* olivas commented:

> Milestone June 2017 deleted


* olivas changed milestone from "June 2017" to ""
